curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efae0543cbd2bbdab94e333.md

```diff
@@ -35,9 +35,15 @@ The Cats `img` element should be nested in the `figure` element.
 
 ```js
 const catsImg = document.querySelectorAll('figure > img')[1];
```

Comment:
```suggestion
const catsImg = document.querySelectorAll('figure > img')?.[1];
```

Reply:
that's not needed, it returns an emmpty nodelist if there are no matches, it is not a point of failure here